### DISCUSSION TITLE
Error occurred prerendering page /posts/.DS_Store when next build

### DISCUSSION THREAD

**Discussion by zhaiduo** [vercel/next.js#15573] [Help] [answered]
Error occurred prerendering page "/posts/.DS_Store". Read more: https://err.sh/next.js/prerender-error:
Error: ENOENT: no such file or directory, open '~/nextjs-demo/posts/.DS_Store.md

Does anyone encounter this building problem?

I've read the doc on /next.js/prerender-error, but it seems no help.

"next": "9.3.5"
MacOS Catalina 10.15.5
node: 10.17.0

**Comment by zhaiduo** [comment, marked as answer]
Anyway I touched .DS_Store.md and .DS_Store. It works.

**Reply by iamdanil** [reply]
Have the same problem. How did you fix this?

**Reply by perryraskin** [reply]
This is strange... why would there need to be `.DS_Store.md` file...? I'm having this issue as well, and adding the `.DS_Store.md` file is just causing other issues...

**Comment by hubertwasier** [comment]
try rm **/.DS_Store before building

**Reply by alexstyl** [reply]
this did the trick for me. thanks :+1:

**Reply by moonkorea00** [reply]
thanks @hubertwasier

solution also works for :
"next": "^13.0.6"
node : v16.14.0
macOS Monterey v12.1

**Reply by KayDrechsler** [reply]
Also did the job for me. Thanks @hubertwasier ! 👏🏻
